### PR TITLE
 Add Symfony 6 compatibility

### PR DESCRIPTION
**Pull request by jdecool**
## Why?

Symfony 6 has just been release.

## How?

Allow Symfony 6 usage in the `composer.json`.

### CODE REVIEW

**Review by lnahiro — APPROVED** [review]

**Review by bestpostmaster — APPROVED** [review]

**Review by thibaudgirard — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -21,17 +21,17 @@
         "psr/container": "^1.0",
         "psr/event-dispatcher": "^1.0",
         "symfony/contracts": "^1.1 || ^2.0",
-        "symfony/expression-language": "^3.4 || ^4.0 || ^5.0",
-        "symfony/framework-bundle":   "^3.4 || ^4.0 || ^5.0",
-        "symfony/property-access": "^3.4 || ^4.0 || ^5.0",
-        "symfony/yaml":   "^3.4 || ^4.0 || ^5.0"
+        "symfony/expression-language": "^3.4 || ^4.0 || ^5.0 || ^6.0",
```
Comment (Oliboy50) [inline composer.json:24]: same [comment for very old symfony versions](https://github.com/BedrockStreaming/GuzzleHttpBundle/pull/75#discussion_r761796423)

**Review by Oliboy50 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (jdecool) [reply composer.json]: Like the other PR, I've removed all unmaintained Symfony versions.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -54,6 +52,12 @@ jobs:
                     # most recent versions
                     - php: 8.0
                       symfony_version: 5.3.*
+                    - php: 8.0
+                      symfony_version: 5.4.*
+                    - php: 8.0
+                      symfony_version: 6.0.*
+                    - php: 8.1
```
Comment (valentin-claras) [inline .github/workflows/ci.yml:59]: Missing symfony version? :thinking:

Comment (jdecool) [reply .github/workflows/ci.yml:59]: All supported Symfony versions are present in at least one test case.

For this one, I leave the default version of Composer.
Do you prefer I specify a version for this one ?

Comment (valentin-claras) [reply .github/workflows/ci.yml:59]: No it's ok, I was wondering what the `symfony/framework-bundle:${{ matrix.symfony_version }}` would do without a value, but it seems to work fine :ok_hand:

**Review by valentin-claras — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jdecool) [reply .github/workflows/ci.yml:59]: The `symfony/framework-bundle` update is conditional:

```yaml
            - name: Install Symfony ${{ matrix.symfony_version }}
              run: composer require "symfony/framework-bundle:${{ matrix.symfony_version }}" --no-update
              if: matrix.symfony_version != ''
```

**Pull request merged**